packages/devextreme-react/src/core/__tests__/component.test.tsx

```diff
@@ -1,137 +1,152 @@
 import * as events from 'devextreme/events';
-
-import { mount, React, shallow } from './setup';
```

Comment:
Can the setup module be removed?
https://github.com/DevExpress/devextreme-react/blob/master/packages/devextreme-react/src/core/__tests__/setup.ts

Reply:
Deleted setup.ts